libs/common/src/response-messages/index.ts

```diff
@@ -59,7 +59,8 @@ export const ResponseMessages = {
             invitationStatusUpdateInvalid: 'Status update is invalid. Request is already',
             resetSamePassword: 'New password should not be the current password',
             resetPasswordLink: 'Unable to create reset password token',
-            invalidResetLink: 'Invalid reset password link or expired'
+            invalidResetLink: 'Invalid reset password link or expired',
+            invalidAccessToken: 'Invalid access token'
```

Comment:
'Unauthorized' or 'Authentication failed' message is also fine. No need to provide technical details to user.

Reply:
Resolved.